packages/actions/actions/UnjailValidator.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback, useMemo } from 'react'
+
+import {
+  Action,
+  ActionComponent,
+  ActionKey,
+  UseDecodedCosmosMsg,
+  UseDefaults,
+  UseTransformToCosmos,
+} from '@dao-dao/tstypes/actions'
+import { makeStargateMessage } from '@dao-dao/utils'
+
+import {
+  UnjailValidatorComponent as StatelessUnjailValidatorComponent,
+  UnjailValidatorIcon,
+} from '../components'
+
+interface UnjailValidatorData {
+  type_url: string
+  value: string
+}
+
+const useDefaults: UseDefaults<UnjailValidatorData> = () => ({
+  type_url: '/cosmos.slashing.v1beta1.MsgUnjail',
+  value: `{
+    "validator_addr": "<your validator address>"
+  }`,
+})
+
+const Component: ActionComponent = (props) => {
+  return <StatelessUnjailValidatorComponent {...props} options={{}} />
+}
+
+const useTransformToCosmos: UseTransformToCosmos<UnjailValidatorData> = () =>
+  useCallback(
+    (data: UnjailValidatorData) =>
+      makeStargateMessage({
+        stargate: {
+          type_url: data.type_url,
+          value: JSON.parse(data.value),
+        },
+      }),
```

Comment:
So this is correct, but we need to also encode the `/cosmos.slashing/v1beta1.MsgUnjail` message in `makeStargateMessage`! If you look at that function you'll see similar examples for other messages.